src/components/StatusIndicator/StatusIndicator.tsx

```diff
@@ -0,0 +1,119 @@
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+import { Icon } from '../../'
+
+import { customPropTypes, UIComponent } from '../../lib'
+import { ComponentVariablesInput, IComponentPartStylesInput } from '../../../types/theme'
+import { Extendable } from '../../../types/utils'
+
+export interface IStatusIndicatorProps {
+  as?: any
+  className?: string
+  size?: number
+  status?:
+    | 'Available'
+    | 'Away'
+    | 'BeRightBack'
+    | 'Busy'
+    | 'DoNotDisturb'
+    | 'Offline'
+    | 'PresenceUnknown'
+  styles?: IComponentPartStylesInput
+  variables?: ComponentVariablesInput
+}
+
+/**
+ * A status indicator is a graphical representation of user's status
+ */
+class StatusIndicator extends UIComponent<Extendable<IStatusIndicatorProps>, any> {
+  static className = 'ui-statusindicator'
+
+  static displayName = 'StatusIndicator'
+
+  static handledProps = ['as', 'className', 'size', 'status', 'styles', 'variables']
+
+  static propTypes = {
+    /** An element type to render as (string or function). */
```

Comment:
could you, please, introduce empty lines between each of the 'prop type + comment' section - so that it would be easier to read this code